charts/cloudserver-front/values.yaml

```diff
@@ -29,6 +29,14 @@ image:
   tag: 0.2.1
   pullPolicy: IfNotPresent
 
+env: {}
+  # If you want to specify additional environmental variables, like http proxy
+  # endpoints, remove the curly braces after 'env:' and the variables below.
+  # http_proxy:
```

Comment:
Not fond of this approach. In the most likely case, one would need to set **4** different custom variables, all with the value, which leaves a lot of room for error (typos, whatnot).

Please reconsider.

Reply:
Technically, the user would only need to set one of each `http` and `https`, as the code checks for both all lowercase and uppercase

Reply:
@dora-korpar In that case, I guess only 2 examples should be provided.
@xaf-scality What do you think?

Reply:
I agree with @dora-korpar . We should allow setting only `HTTP_PROXY` or `HTTPS_PROXY` and case management will be handled by CloudServer.

Reply:
My point was that this allows to set arbitrary environment variables, and happens to document 2 important ones. In the ticket I described having 2 dedicated configuration options, where even setting only one of them would be sufficient in most cases to have the system running.

The fact that this one (or 2) options are rendered into however many environment variables using whatever names for the container environment (i.e. in the template logic) is irrelevant at that point.

Reply:
This approach doesn't look the best for me too. The approach @NicolasT suggested was better, but it introduced unnecessary templating. However, I later figured a simple workaround using '{}' braces. Updating the PR now.